extjsdk/src/main/java/io/vantiq/extjsdk/ExtensionWebSocketListener.java

```diff
@@ -271,6 +271,8 @@ public void onOpen(WebSocket webSocket, Response response) {
         log.info("WebSocket open");
     }
 
+    private final String CONNECT_ERROR = "io.vantiq.sourcemgr.connectExtension";
```

Comment:
This might be better placed in ExtensionServiceMessage

Reply:
Will do.

Reply:
Actually, without the check in the next comment this constant isn't necessary.